gtest/src/system.rs

```diff
@@ -42,7 +42,11 @@ impl System {
     }
 
     pub fn init_logger(&self) {
-        let _ = Builder::from_env(Env::default().default_filter_or("gwasm=debug"))
+        self.init_logger_with_default_filter("gwasm=debug")
+    }
+
+    pub fn init_logger_with_default_filter<'a>(&self, default_filter: impl Into<Cow<'a, str>>) {
```

Comment:
Isn't it just `filter`, not default?
```suggestion
    pub fn init_logger_with_filter<'a>(&self, filter: impl Into<Cow<'a, str>>) {
```

Reply:
Not sure this is a proper name for the func. This filter is taken into account only in cases when it is not specified by other means. For the underlaying method this param is called `default_filter`.
What do you think about naming it `init_logger_with_fallback_filter`?